### DISCUSSION TITLE
How Do You Print a Pattern to Size?

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#6965] [Forum]
Hello, I am new and wanted to make a cathrin pattern with my measurements, however, I have no idea how to print it to size on my printer and it keeps getting squished.

[![User false](https://backend3.freesewing.org/users/false/card)](https://next.freesewing.org/users/false)

**Comment by BenJamesBen** [comment]
1. When you generate a PDF of the pattern, make sure that the correct **Paper Size** is selected for your printer.
_(The default paper size that FreeSewing uses is A4. If you try to print an A4-sized PDF onto Letter-sized paper then printer might stretch or squish the printed pattern which is something you want to avoid.)_
2. Before printing out the entire pattern, print just the first page, the one with the title box that says "The (white) inside of this box should measure _x_ cm. The (black) outside of this box should measure _x_ cm." Then, make sure that this box is the correct dimensions on the printed page to confirm that things are getting printed at the right size.
_(If the box is stretched or squished, work to fix the issue before printing out the entire pattern.)_
3. If you use are using a program like Adobe Acrobat Reader to print the PDFs to your printer, check the print settings selected to make sure that you are printing at "Actual Size" or 100% scale.
_(Avoid settings like "Fit", "Fit to page", or "Shrink oversized pages".)_
